xds/internal/httpfilter/fault/fault_test.go

```diff
@@ -465,15 +465,8 @@ func (s) TestFaultInjection_Unary(t *testing.T) {
 
 	fs, nodeID, port, cleanup := clientSetup(t)
 	defer cleanup()
```

Comment:
Could we move these two lines to be inside `t.Run()` as well.

Reply:
I tried that initially but that actually breaks things or at least would make them a lot slower.